### PR TITLE
Added internal java solution for measuring traffic

### PR DESCRIPTION
**Pull request by sathu-beep**
fixed a small error in a pull request that went unnoticed 👀 that I rediscovered today

### CODE REVIEW

**Review by ar492 — APPROVED** [review]
Looks good; I trust the code works. however, I don't understand why there's no explanation / why you don't just put it in the "user solutions" section or whatever it's called.

**Comment by maggie-j-liu** [comment]
#1856 exists...
i think keep that one since it's for cpi video classes

**Review by ar492 — CHANGES_REQUESTED** [review]
https://github.com/cpinitiative/usaco-guide/pull/1856

File: solutions/usaco-917.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
```
Comment (dongliu0426) [inline solutions/usaco-917.mdx:1]: can you delete this file then?

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

**Comment by dongliu0426** [comment]
> #1856 exists...
> i think keep that one since it's for cpi video classes

**Pull request closed without merging**